src/main/java/uk/ac/sanger/sccp/stan/config/MlwhConfig.java

```diff
@@ -0,0 +1,23 @@
+package uk.ac.sanger.sccp.stan.config;
+
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.context.annotation.Configuration;
+
+import java.sql.*;
+
+/**
+ * @author dr6
+ */
+@Configuration
+public class MlwhConfig {
+    @Value("${spring.mlwh.url}")
+    String url;
+    @Value("${spring.mlwh.username}")
+    String user;
+    @Value("${spring.mlwh.password}")
+    String password;
+
+    public Connection getConnection() throws SQLException {
+        return DriverManager.getConnection(url, user, password);
+    }
```

Comment:
Just to confirm? Is this configuration to set STAN Dnap studies from mlwh?

Reply:
Yes